liquidcore/config/system.py

```diff
@@ -8,6 +8,17 @@ def get_configuration():
         for item in models.Setting.objects.all()
     }
 
+    client_keys = [
+        {'id': str(key.id), 'revoked': key.revoked}
+        for key in models.VPNClientKey.objects.all()
+    ]
+
+    vpn_client_config_file = settings['vpn_client_config_file']
+    if vpn_client_config_file:
+        vpn_client = {'config_file': vpn_client_config_file}
```

Comment:
Instead of building the dict here, we could create it in `views.py`. That would let us add more metadata when needed (like upload time and user).

Additionally, I think that `config_file` should have the word `content` in its name, because it's a string with the file content.

Reply:
Agreed about `_content` in the variable name. I think it's OK to build the dict here, views.py is in this same repo, so we can always refactor and write migrations.